meson.build

```diff
@@ -1,7 +1,7 @@
 project('colord', 'c',
   version : '1.4.5',
   license : 'LGPL-2.1+',
-  meson_version : '>=0.46.0',
+  meson_version : '>=0.52.0',
```

Comment:
Is this actually required to make the above code work?  I would _guess_ not.  It's better to put off bumping up the meson dependency as long as possible in my mind so it's easier to backport and do stable updates in older distros and the like.

Reply:
meson complains that the features I'm using is in 0.52.0 and above only

Reply:
```
WARNING: Project specifies a minimum meson_version '>=0.46.0' but uses features which were added in newer versions:
 * 0.47.0: {'dict'}
 * 0.50.0: {'install arg in configure_file'}
 * 0.52.0: {'environment positional arguments'}
```

Reply:
ah OK.

Reply:
Yeah definitely then needs bumping, thanks for confirming.